atox/src/main/res/layout/contact_list_view_item.xml

```diff
@@ -2,19 +2,28 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:app="http://schemas.android.com/apk/res-auto"
         xmlns:tools="http://schemas.android.com/tools"
+        xmlns:custom="http://schemas.android.com/apk/res-auto"
```

Comment:
The namespace's already here, but named `app`. This goes for all files the custom namespace was added to.

Reply:
Right! And now I have auto-complete for the attributes!